BUILD on Mac M1, Return ERROR: ld: symbol(s) not found for architecture arm64

OS version = macOS Big Sur 11.6
CPU = Apple M1

$go get github.com/lukeroth/gdal
\# github.com/lukeroth/gdal
Undefined symbols for architecture arm64:
  "_CPLGetConfigOption", referenced from:
      __cgo_20816c41b399_Cfunc_CPLGetConfigOption in _x003.o
     (maybe you meant: __cgo_20816c41b399_Cfunc_CPLGetConfigOption)
  "_CPLSetConfigOption", referenced from:
      __cgo_20816c41b399_Cfunc_CPLSetConfigOption in _x003.o
     (maybe you meant: __cgo_20816c41b399_Cfunc_CPLSetConfigOption)
  "_GDALAddBand", referenced from:
      __cgo_20816c41b399_Cfunc_GDALAddBand in _x003.o
     (maybe you meant: __cgo_20816c41b399_Cfunc_GDALAddBand)
  "_GDALAllRegister", referenced from:
      __cgo_20816c41b399_Cfunc_GDALAllRegister in _x003.o
     (maybe you meant: __cgo_20816c41b399_Cfunc_GDALAllRegister)
..................
  "_VSIFOpenL", referenced from:
      __cgo_20816c41b399_Cfunc_VSIFOpenL in _x003.o
     (maybe you meant: __cgo_20816c41b399_Cfunc_VSIFOpenL)
  "_VSIFReadL", referenced from:
      __cgo_20816c41b399_Cfunc_VSIFReadL in _x003.o
     (maybe you meant: __cgo_20816c41b399_Cfunc_VSIFReadL)
  "_VSIReadDirRecursive", referenced from:
      __cgo_20816c41b399_Cfunc_VSIReadDirRecursive in _x003.o
     (maybe you meant: __cgo_20816c41b399_Cfunc_VSIReadDirRecursive)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

--- a/c_darwin_arm64.go
+++ b/c_darwin_arm64.go
@@ -1,0 +1,9 @@
+// +build darwin,arm64
+
+package gdal
+
+/*
+#cgo pkg-config: gdal
+#cgo LDFLAGS: -Wl,-undefined,dynamic_lookup
+*/
+import "C"

